README.md

```diff
@@ -49,6 +49,15 @@ cmake ..
 make
 ```
 
+### Creating Packages/Installers using CMake/CPack
+
+Packages for certain Linux distributions can be automatically created using `cpack`. After compilation run:
+```
+cd build
+cpack -G <generator> -D CPACK_PACKAGE_CONTACT="<your contact>"
+```
+For more information see the [CPack documentation](https://cmake.org/cmake/help/latest/manual/cpack.1.html).
+
```

Comment:
I don't think it should be in the readme before I haven't tested it fully yet and can be "officially supported". The changes themselves lgtm though.